netlogo-gui/src/main/lab/gui/ProgressDialog.scala

```diff
@@ -20,13 +21,14 @@ private [gui] class ProgressDialog(dialog: java.awt.Dialog, supervisor: Supervis
   private val totalRuns = protocol.countRuns
   private val progressArea = new JTextArea(10 min (protocol.valueSets.size + 3), 0)
   private val timer = new Timer(PeriodicUpdateDelay.DelayInMilliseconds, periodicUpdateAction)
-
-  private var updatePlots = true
+  private var updatePlots = false
```

Comment:
there's no specific reason to make ``updatePlots = false`` , it gets updated in ``setPlotsAndMonitors(status : Boolean)`` method when it is called in ``start()`` method in Supervisor.scala code.

Reply:
Right, so why is this change needed? In general, we want pull requests to limit changes which aren't necessary so that we can focus on what changes *are* necessary. If it's important that this be `false`, that's fine, but that isn't a requirement here, so we would prefer not to have this change.

Reply:
alright, will modify to its orignal state.